theme6/source/stylesheets/site.css.scss

```diff
@@ -0,0 +1,24 @@
+body {
+  background-color: #fbc547;
+  color: #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.